.github/workflows/tapioca-update.yml

```diff
@@ -0,0 +1,36 @@
+name: Tapioca Update
+on:
+  pull_request:
+    paths:
+      - .github/workflows/tapioca.yml
+
+  schedule:
+    - cron: '0 0 */3 * *' # Every three days
+
+env:
+  HOMEBREW_SORBET: 1
+jobs:
+  update:
+    runs-on: [ubuntu-latest, macOS-latest]
```

Comment:
Why do we need to run this on both macOS and Linux? 🤔

Reply:
Also note that this syntax won't work, as it has to be a matrix.

Reply:
Thank you for the heads up! Just pushing a fix.